src/ar_bridge.erl

```diff
@@ -81,6 +81,20 @@ add_remote_peer(PID, Node) ->
 			PID ! {add_peer, remote, Node}
 	end.
 
+-ifdef(DEBUG).
+%% Do not filter out loopback IP addresses with custom port in the debug mode
+%% to allow multiple local VMs to peer with each other.
+is_loopback_ip({_, _, _, _, Port}) -> Port == ar_meta_db:get(port).
```

Comment:
This whole is_loopback_ip and how we use it is weird. I think what we're trying to do is to not add self as a remote. It would make sense to call the function `is_peer_self` and do some adjustments. Let's talk about this.

Reply:
✅